scripts/google.test

```diff
@@ -6,6 +6,11 @@ server=www.google.com
 
 [ ! -x ./examples/client/client ] && echo -e "\n\nClient doesn't exist" && exit 1
 
+if test -n "$WOLFSSL_NO_GOOGLE_TEST"; then
```

Comment:
Any reason we can't change this to use the existing `WOLFSSL_EXTERNAL_TEST` env variable?

Reply:
No reason at all.  Will make the change.